webapp/src/js/controllers/inbox.js

```diff
@@ -381,11 +381,9 @@ var _ = require('underscore'),
     Changes({
       key: 'inbox-facilities',
       filter: function(change) {
-        var hierarchyTypes = ContactSchema.getPlaceTypes().filter(function(pt) {
-          return pt !== 'clinic';
-        });
-        // check if new document is a contact
-        return hierarchyTypes.indexOf(change.doc.type) !== -1;
+        return change.doc.type !== 'clinic' &&
+               change.doc.type !== 'person' &&
+               ContactTypes.includes(change.doc);
```

Comment:
Maybe we should exclude person types and leaf place types? Maybe at least person types?

Reply:
Definitely. Thanks.